.ci/jenkins/build-int-test/Jenkinsfile

```diff
@@ -11,7 +11,7 @@ pipeline {
     }
     parameters {
         // TODO: update default value as newer PGE's are added
-        string(name: 'DOCKER_IMAGE_SUFFIXES', defaultValue: 'dswx_hls,cslc_s1,rtc_s1',
+        string(name: 'DOCKER_IMAGE_SUFFIXES', defaultValue: 'dswx_hls,cslc_s1,rtc_s1,dswx_s1',
```

Comment:
Note for @david-inglish, I think this will need to be updated for your DISP-S1 int test branch as well